code/modules/reagents/chemistry/reagents.dm

```diff
@@ -85,9 +85,12 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	/// The affected bodytype, if the reagent damages/heals bodyparts (Brute/Fire) of an affected mob.
 	/// See "Bodytype defines" in /code/_DEFINES/mobs.dm
 	var/affected_bodytype = BODYTYPE_ORGANIC
-	/// The affected biotype, if the reagent damages/heals generic damage (Toxin/Oxygen) of an affected mob.
+	/// The affected biotype, if the reagent damages/heals toxin damage of an affected mob.
 	/// See "Mob bio-types flags" in /code/_DEFINES/mobs.dm
 	var/affected_biotype = MOB_ORGANIC
+	/// The affected respiration type, if the reagent damages/heals oxygen damage of an affected mob.
+	/// See "Mob bio-types flags" in /code/_DEFINES/mobs.dm
+	var/affected_respiration_type = RESPIRATION_OXYGEN
```

Comment:
this change is real expansive, have a hard time trusting it. that and the forced = true smells weird

Reply:
I'm a bit worried about side effects of this, like reagents that should have healed X lung in the past not working